notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29b161d0-8144-4b3f-88fe-deb6abd21f1c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Meter Reader with PaddleOCR\n",
+    "This notebook shows how to create a meter reader with OpenVINO Runtime. We use the pre-trained [PP-OCR](https://github.com/PaddlePaddle/PaddleOCR) to build up a inference task pipeline:\n",
+    "\n",
+    "1. Config the screen area of the meter reader.\n",
```

Comment:
Suggest you can use Grammarly or any other tool to fix all typos and grammar issues in the notebook, for example: config is should be a verb here.